packages/js/product-editor/src/components/tabs/tabs.tsx

```diff
@@ -102,10 +97,10 @@ export function Tabs( { selected, onChange }: TabsProps ) {
 		}
 	}
 
-	function renderFills( fills: readonly ( readonly ReactElement[] )[] ) {
+	function renderFills( fills: ReactNode ) {
 		return (
 			<TabFills
-				fills={ fills }
+				fills={ fills as readonly ( readonly ReactElement[] )[] }
```

Comment:
Hmm given this will likely be removed or altered, I am bound to let this slide.
But ideally we should update `TabFills` to also receive `ReactNode` and make use of `Children.toArray`.
Although the `sortFillsByOrder` is a pretty well duplicate of the one available in the components package: https://github.com/woocommerce/woocommerce/blob/d32cc4f22dac5020abd92765a0573253d35917ca/packages/js/components/src/utils.tsx#L93
So we can either mimic that or use that one.

Anyway, no hard feelings if you decide to ignore :)

Reply:
I'm fine with both solutions, at the same fine, I'm more inclined to keep the code as it is given that we already made multiple changes across the codebase (especially given that we will remove or change it)